pkg/kubectl-argo-rollouts/cmd/status/status.go

```diff
@@ -0,0 +1,123 @@
+package status
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/info"
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/options"
+	"github.com/argoproj/argo-rollouts/pkg/kubectl-argo-rollouts/viewcontroller"
+	"github.com/spf13/cobra"
+)
+
+const (
+	statusLong = `Watch the progress of a rollout until it is done. The return will be success if
+the Rollout ends as Healthy and error if the Rollout ends degraded or if the
+timeout is hit before the progress finishes.`
```

Comment:
I'm not a fan of `the progress finishes`

Can you explain what you're trying to say?

Note that this uses `hit` as opposed to `within` (`hit` is vaguely ok, `within` as noted below isn't, although `hit` isn't really great)

Reply:
That is indeed not ideally worded.

If the Rollout is still either in Progressing or Paused by the time the timeout is reached the commands returns with an error.

This is the same behaviour that happens in `kubectl rollout status` when a Deployment doesn't finish its rollout before the specified timeout.

Reply:
`Watch rollout until it finishes or if a timeout is specified the timeout is exceeded`?
`Returns success if the rollout is healthy upon completion and an error otherwise`?

Reply:
There is a default timeout specified, but I think that could work:

> Watch rollout until it finishes or the specified timeout is exceeded. Returns success if the rollout is healthy upon completion and an error otherwise.

Reply:
This sounds pretty good, but, if there's a default timeout, then drop `specified`.